src/groupeddataframe/grouping.jl

```diff
@@ -429,9 +461,11 @@ function combine(f::Any, gd::GroupedDataFrame)
         return gd.parent[1:0, gd.cols]
     end
 end
+
 combine(gd::GroupedDataFrame, f::Any) = combine(f, gd)
-combine(gd::GroupedDataFrame, f::Pair...) = combine(f, gd)
-combine(gd::GroupedDataFrame, f::Pair) = combine(f, gd)
+
+combine(gd::GroupedDataFrame, f::Union{Pair, AbstractVector{<:Pair}}...) =
+    combine(reduce(vcat, f), gd)
 
 function combine(gd::GroupedDataFrame; f...)
```

Comment:
It is not clear for me how you handle the case `combine(gd, (a=1=>length, b=2=>length))`.

Reply:
OK - I see this is not this case (it is `;` instead of `,` in the signature 😄)